src/sciline/_provider.py

```diff
@@ -27,7 +27,7 @@
 """Callable that can be converted to a provider."""
 
 ProviderKind = Literal[
-    'function', 'parameter', 'series', 'table', 'sentinel', 'unsatisfied'
+    'function', 'parameter', 'series', 'table_label', 'sentinel', 'unsatisfied'
```

Comment:
Because this is what it's called in the code: https://github.com/scipp/sciline/blob/f24a62f51bdee6ec4983b0f1beb90bd5c24f54c3/src/sciline/pipeline.py#L421

Reply:
Yes, but shouldn't that be renamed?

Reply:
Don't know. Pipeline seems to use 'label' and 'index' interchangeably for tables. So we could rename it to index?

Reply:
No, this is two different things. If you rename it to index it would mean an entry of the index column:

- A param table has an "index" column, and one or more data columns. Columns are labeled using a type.
- The index is a bit like an index in a `pandas.DataFrame` (and it also has a column label, the type of the index).
- A cell in the table is identified using an index value, i.e., row index (an index, in the alternate meaning of the word index, i.e., an entry of the index column) and a column label.

Reply:
You renamed the method to `table_cell`, but here it is still `table_label`.